cbor-schema/plt.cddl

```diff
@@ -0,0 +1,156 @@
+; This CDDL schema defines the CBOR encodings used by protocol-level tokens (PLTs) in Concordium.
+; In particular, it defines the types token-holder-transaction, token-governance-transaction, and
+; token-initialization-parameters, which are used as parameters for the token module.
+
+; A token holder transaction body. This is a list of token-holder operations, executed in order.
+; The transaction is executed in the context of a sender account and a designated token.
+token-holder-transaction = [ * token-holder-operation ]
+
+; A token-holder operation. Currently, only token transfers are supported.
+token-holder-operation = token-transfer
+
+; A token transfer operation. This transfers a specified amount of tokens from the sender account
+; (implicit) to the recipient account.
+token-transfer = { 
+    ; The operation type is "transfer".
+    "type": "transfer",
+    ; The amount of tokens to transfer.
+    "amount": token-amount,
+    ; The recipient account.
+    "recipient": token-receiver,
+    ; An optional memo.
+    ? "memo": memo
+}
+
+; We represent token amounts as a decimal fraction.
+; The range and precision may be constrained by the token itself.
+token-amount = decfrac
```

Comment:
I feel like we might shoot ourselves in the foot with this representation of token amounts. In the protobuf types, token amounts are represented with the integer amount and a number of decimals. This allows the token amount to have a fractional number representation while also allowing seemless serialization and deserialization back into the same token amount representation.

Maybe we can use a tag here as well to ensure correct interpretation of the value?

Reply:
My assumptions about `decfrac` were wrong.. sorry 😅